### ISSUE TITLE
Website does not tell me why I'd want your product

### ISSUE THREAD

**Issue by yam655** [jellyfin/jellyfin#2396]
**Describe the bug**

Your website does not sell your product.

To find out about the product and why I would want to use it, I had to do a Google search on "Emby" -- the thing it used to be.

**To Reproduce**
1. Go to https://jellyfin.org/
2. Click on 'Learn More'
3. Note there's no compelling reason to use the product there
4. Click the FAQ
5. Note the lack of "What is Jellyfin?" and "Why would I want to use Jellyfin?"
6. Accept that Jellyfin is defined by what it is not: Emby. 
7. Click the "about page" in the "Why fork Emby?" question

What does the About page actually say about your product? Absolutely nothing. Here, I break it down:

~Jellyfin is a Free Software Media System that puts you in control of managing and streaming your media.~ (Meaningless marketing blather.) ~It is an alternative to the proprietary Emby and Plex, to provide media from a dedicated server to end-user devices via multiple apps. Jellyfin is descended from Emby's 3.5.2 release and ported to the .NET Core framework to enable full cross-platform support.~ (References Emby, which I know nothing about.) ~There are no strings attached, no premium licenses or features, and no hidden agendas: just a team who want to build something better and work together to achieve it. We welcome anyone who is interested in joining us in our quest!~ (Standard open source.)

~Jellyfin seeks to continue development of the original Emby project with a Free Software ethos.~ (I still no nothing of Emby.) ~It is committed to bringing all its users access to the best possible Media System, developed entirely by a community of volunteers who contribute code, documentation, translations, and support to the project.~ (Open source. What's a "Media System"?)

~The Jellyfin project was started in early December 2018 as a result of Emby's decision to take their 3.6 release closed-source, as well as various philosophical differences with the core developers. Jellyfin seeks to be the free software alternative to Emby and Plex to provide media management and streaming from a dedicated server to end-user devices.~ (Again references Emby, which I know nothing about.)

**Expected behavior**

I expected something that tried to convince me of why I should use the product. 

What is it? What can I do with it? Does it have any compelling features against the competition?

Basically, have you seen any of the following pages:
- https://www.plex.tv/
- https://kodi.tv/about
- https://emby.media/about.html

**Logs**

n/a

**Screenshots**

n/a

**System:**

n/a (only a potential user)

**Additional context**

**Comment by Shadowghost** [comment]
Anything constructive in this issue report (yes the passive-agressivity is intentional)? For me that sounds more like a rant about how bad you think the homepage is without any constructive help (no, linking other media server's hompages does not count - do you really think the dev's don't know them?).
Jellyfin does not want to "sell" you anything btw. This is an open source project, all people contributing do it in their freetime and for free and for a common goal: make the curren application better.
Good thing about open source: You can contribute and change what you think could be done better, e.g. propose an idea (or in this case: a mockup) specifically what and how it can be done. Not just some "Here look that's better and that one too" - this will help nobody.
Issue reports are meant as a neutral constructive place where everyone can give comments and help one out - but do you really expect that someone will take you seriously (even if the gist of your rant isn't false at all) with such kind of attitude?

**Comment by joshuaboniface** [comment]
You have a point. Most of this was written early in the fork, when we were "A fork of Emby" first - we expected most of our users to be familiar with Emby as a product and sought to emphasize that relationship as a means of obtaining users. Things have definitely changed over the year since, where we've grown into a solution of our own, and your criticism is valid (if a bit passive-aggressive - I wrote that "[m]eaningless marketing blather").

I'm interested in your proposed solutions. What would you write instead? How should we improve this? Most collaborators were (and some still are) users of Emby and Plex, so our perspective is skewed.

**Comment by AndreasGB** [comment]
First of all, welcome and thank you for your feedback, @yam655! I agree with you, that the copywriting could be updated, please contribute if you have experience in that area.

As a side comment, you claim that everyting being free is standard for open source projects. This is not true. A lot of open source projects, for example nginx or influxDB develop their projects as an open source core application, where additional features needs to be purchased from them.

**Comment by joshuaboniface** [comment]
>A lot of open source projects, for example nginx or influxDB develop their projects as an open source core application, where additional features needs to be purchased from them.

Indeed, and this was a big factor in the wording we chose. We want to emphasize that Jellyfin is not, nor will ever be, an "Open Core" application and that it is volunteer-only.

**Comment by yam655** [comment]
I didn't really know where to put in a report for the website.

I used the tools and the framework that was available. Since I couldn't find a place to comment, I went to Github. Since there was no place to comment without creating a bug report, I created a bug report. The bug report had an explicit structure it wanted followed, so I followed that structure.

If there were any other way to comment on the website, I would have used it. I wasn't trying to be harsh or passive-aggressive, so much as highlighting the total absence of information relating to what the product is or does. What is it? What does it do? I have no clue. 

I was trying to be helpful. It's a GitHub ticket, so it's easy enough to close/will-not-fix if it isn't helpful. I admit, it was mostly a drive-by ticket. I have no emotional investment in the outcome of the ticket.

I don't use any sort of media server product, though I've been looking at them off-and-on for decades.  It's not that I've grudgingly been using a media server I don't really like. It's that I've not found a product that seems worth the time to install it.

Sometimes I think I might want one.

- Why do people love it? What does it do?

- How does it compare to systems that ship with operating systems? (Media sharing, simple disk sharing etc.)

- What makes the product worth the installation time?

- How does it compare to other products?

**Comment by anthonylavado** [comment]
I'm not going to comment on the rest of the discussion (except for this). I will focus on direct concerns brought up about usability, discovery, and text.

 The "Learn More" link was originally supposed to scroll down the page, to review more information about the program. I've restored that functionality here: https://github.com/jellyfin/jellyfin-blog/pull/20

It's not the most detailed set of sections right now, and I have sketches/notes on how to replace it and enhance it.

**Comment by yam655** [comment]
I'll add this last bit of perspective.

My web browser hides my scroll bar. I was following visible links. I did not even think about trying to scroll the page. 

Had I been able to find the information that is now available, I would not have made the ticket.

**Comment by joshuaboniface** [comment]
Don't worry I want to keep this open for further discussion - tone is hard in text ;-)

If you have any ideas, we take PRs on the jellyfin/jellyfin.github.io repo, if not I'll try to incorporate some of the feedback into a front-page rewrite. You do have my thanks.
